lib/cadet/updater/updater.ex

```diff
@@ -1,4 +1,4 @@
-defmodule Cadet.Public.Updater do
+defmodule Cadet.Updater.Public do
```

Comment:
I think you meant the file name to be `lib/cadet/updater/public.ex`?

Reply:
oh no